lib/modules/subredditManager.js

```diff
@@ -1189,6 +1195,36 @@ function toggleSubredditDropdown(e: Event) {
 	}
 }
 
+function addFilterSearch(theBody) {
+	const tableHead = document.createElement('thead');
+	const tableRow = document.createElement('tr');
+	const tableCol = document.createElement('td');
+	const search = document.createElement('input');
+
+	tableCol.setAttribute('colspan', '3');
+	search.setAttribute('placeholder', 'Filter subreddits...');
+
+	tableCol.appendChild(search);
+	tableRow.appendChild(tableCol);
+	tableHead.appendChild(tableRow);
+	// focus on open (delay to allow render)
+	setTimeout(() => { search.focus(); }, 100);
```

Comment:
`requestAnimationFrame` would probably be better here, since it'll be called immediately before the first paint after this is injected.

Still a bit sketchy, though I can't really think of a better option. (`autofocus` seems to only work once)